modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,28 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.2
```

Comment:
- should `2.4.2` use the `SMProductVersion` attribute that is defined in the `common-atributes.adoc` file?